src/app/modules/group/components/group-log-view/group-log-view.component.ts

```diff
@@ -62,17 +81,41 @@ export class GroupLogViewComponent implements OnChanges, OnDestroy {
 
   refresh(): void {
     this.refresh$.next();
+    this.resetRows();
   }
 
-  private getData$(groupId?: string): Observable<Data> {
-    return this.activityLogService.getAllActivityLog(groupId).pipe(
-      map((data: ActivityLog[]) => ({
+  private getData$(): Observable<Data> {
+    return this.datapager.list$.pipe(
+      map(fetchData => ({
         columns: this.getLogColumns(),
-        rowData: data
+        rowData: fetchData.data ?? [],
+        isFetching: fetchData.isFetching,
+        hasError: fetchData.isError,
```

Comment:
If I get it right, `fetchData` is already a fetch state (btw, the naming is not correct).
`FetchState` is important data structure in app, it already abstract the fact that a fetch may be in loading, ready or error and may have some data. 
So basically, here, you unwrap it (losing some part of its interesting properties)... but basically it probably makes more sense to use this fetchState directly and let the template deal with it.